src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model.mdx

```diff
@@ -147,8 +147,10 @@ The user model is independent of your pricing plan. For how user model relates t
 The new user model offers [many benefits](https://newrelic.com/blog/how-to-relic/user-management-and-access-security). Here are some feature impacts of being on the New Relic One user model that may be surprising or unintuitive to our existing customers:
 
 * More separation of organizations/accounts: If you have an email address associated with multiple New Relic accounts, this may mean you need to log out and log back in. [Learn more about account access.](/docs/accounts/accounts-billing/general-account-settings/factors-affecting-access-features-data/#account-access) 
-* Adding accounts: [Pro and Enterprise edition](https://newrelic.com/pricing) can have more than one account in their organization. Currently users in these organizations cannot, on their own, add accounts. They must get help from their New Relic account representative. To learn about adding users to accounts, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
-* Alert notifications: Users on the new model won’t show up as an option when [choosing users to receive alert notifications](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/#user). The result of this is that these users will not get [New Relic mobile app push notifications](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps/). As a workaround, you can instead use their [email address](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/#email) as the notification channel.
+* Adding accounts: [Pro and Enterprise edition](https://newrelic.com/pricing) can have more than one account in their organization. Currently users in these organizations cannot, on their own, add accounts. They must get help from their New Relic account representative. To learn about adding users to accounts, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts). 
+* Alerts-related impacts: 
+  * Policy actions: when viewing an alert policy, it displays the user that made the last modification. For users on the new model, this displays their user ID and not their user name. For example, it might say `Last modified Sep 7, '20 4:26 pm <var>by user 1234567</var>` instead of `by Thomas Johnson`.
```

Comment:
When, capital W

Reply:
Also, not sure about using might. I'd go for "For example, it will read 'Last modified..."